lib/agent/actions/fileretrieval/index.js

```diff
@@ -0,0 +1,124 @@
+"use strict";
+
+//////////////////////////////////////////
+// Prey JS FileRetrieval
+// (C) 2016 Prey, Inc.
+// by Mauricio Schneider and Javier Acuña - http://preyproject.com
+// GPLv3 Licensed
+//////////////////////////////////////////
+
+var fs = require('fs'),
+    path = require('path'),
+    mime = require('mime'),
+    needle = require('needle'),
+    join = require('path').join,
+    api = require('./../../plugins/control-panel/api'),
+    files = require('./files_storage'),
+    EventEmitter = require('events').EventEmitter;
+
+var common   = require('./../../common'),
```

Comment:
all require calls should be in one var group and non require calls should be in a different var group

```
// First group
var a = require('a'),
  b = require('b');

// Second group
var something_with_a = a.foo(),
  something_with_b = b.bar();
```